src/nvim/api/ui.c

```diff
@@ -86,6 +86,7 @@ void nvim_ui_attach(uint64_t channel_id, Integer width, Integer height,
   ui->put = remote_ui_put;
   ui->bell = remote_ui_bell;
   ui->visual_bell = remote_ui_visual_bell;
+  ui->default_colors_set = remote_ui_default_colors_set;
```

Comment:
maybe just `colors_set` ?

Reply:
Wouldn't that be too similar to `set_highlight`? I would like some qualifier like "default" or "global" to make the distinction more clear.

Reply:
how about `colors_init` then. It seems weird to start putting "default" in API function/event names.

Reply:
Or `colors_configure` -- that might be a good convention for cases where the client is expected to manage state.

Reply:
Not sure I like it, all highlights are "configured". And it is not only initialization, these values is continously used and can be changed. How about calling them what they are called elsewhere, `normal_colors_set` ?

Reply:
default_colors_set is more intuitive than normal_colors_set, let's stick with that.